example/ios/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem "cocoapods", "1.16.0"
```

Comment:
Please move Gemfile to the project root.

Rationale: In ci.yaml, we did not explicitly set up ruby, install gems, and run cocoapods. But we should really do that. Having Gemfile and Gemfile.lock at the project enables us to use ruby/setup-ruby, as it will pick up these two files by default.

Reply:
Moved Gemfile to root